recipes/fABBA/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "fABBA" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bf0672c2d0be449627380f51a7e695ddd43bf548a292de9dcfe8562e256ff95a
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+    
+requirements:
+  build:
+    - python >=3.6
+    - pip
+    - numpy 
+    - cython
+
+  run:
+    - python >=3.6
+    - pandas
+    - requests
+    - numpy
+    - tqdm
+    - scikit-learn
+    - scipy ==1.7.2
+    - matplotlib-base
```

```suggestion
requirements:
  build:
    - {{ compiler('c') }}
  host:
    - python
    - pip
    - numpy 
    - cython
  run:
    - python
    - pandas
    - requests
    - {{ pin_compatible('numpy') }}
    - scikit-learn
    - scipy >=1.2.1
```
Upstream states these run requirements. No matplotlib or tqdm.

---

I'm also pretty sure that your setup.py is wrong regarding cython. cython is listed in the `install_requires` section, but it should probably be listed in the `setup_requires` section.

---

Thanks so much for pointing out this!